root.tf

```diff
@@ -107,6 +107,40 @@ module "prod_environment_role" {
   tdr_mgmt_account_number = data.aws_ssm_parameter.mgmt_account_number.value
 }
 
+//Shared parameters to store in each environment
+module "intg_environment_paramers" {
```

Comment:
Could put the module name out in full, ie: intg_environment_parameters

Reply:
Oops, that's a typo. Fixed!